.releaserc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I believe this should not be in your PR

Reply:
I agree with Michael, let's please keep those separate